core/src/components/picker-column/test/disabled/picker-column.e2e.ts

```diff
@@ -1,123 +1,75 @@
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
-/**
- * This behavior does not vary across directions.
- */
-configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
-  test.describe(title('picker-column: disabled rendering'), () => {
-    test('should not have visual regressions', async ({ page }) => {
-      await page.setContent(
-        `
-        <ion-picker>
-          <ion-picker-column value="b"></ion-picker-column>
-        </ion-picker>
-
-        <script>
-          const column = document.querySelector('ion-picker-column');
-          column.items = [
-            { text: 'A', value: 'a', disabled: true },
-            { text: 'B', value: 'b' },
-            { text: 'C', value: 'c', disabled: true }
-          ]
-        </script>
-      `,
-        config
-      );
-
-      const picker = page.locator('ion-picker');
-      await expect(picker).toHaveScreenshot(screenshot(`picker-disabled`));
-    });
-  });
-});
-
 /**
  * This behavior does not vary across modes/directions.
  */
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
   test.describe(title('picker-column: disabled items'), () => {
+    // TODO FW-5580 move this to a spec test in picker-column-option
     test('all picker items should be enabled by default', async ({ page }) => {
       await page.setContent(
         `
         <ion-picker>
-          <ion-picker-column></ion-picker-column>
+          <ion-picker-column>
+            <ion-picker-column-option value="a">A</ion-picker-column-option>
+            <ion-picker-column-option value="b">B</ion-picker-column-option>
+            <ion-picker-column-option value="c">C</ion-picker-column-option>
+          </ion-picker-column>
         </ion-picker>
-
-        <script>
-          const column = document.querySelector('ion-picker-column');
-          column.items = [
-            { text: 'A', value: 'a' },
-            { text: 'B', value: 'b' },
-            { text: 'C', value: 'c' }
-          ]
-        </script>
       `,
         config
       );
 
-      const pickerItems = page.locator('ion-picker-column .picker-item:not(.picker-item-empty, [disabled])');
+      const pickerItems = page.locator('ion-picker-column ion-picker-column-option button:not([disabled])');
 
       expect(await pickerItems.count()).toBe(3);
     });
+    // TODO FW-5580 move this to a spec test in picker-column-option
```

Comment:
This is much easier to test using a spec test now, so I'd like to update this in a future PR.